src/main/java/com/github/mauricioaniche/ck/CKMethodResult.java

```diff
@@ -41,13 +44,14 @@
 	//All direct invocations of methods of the same class
 	private Set<String> methodInvocationsLocal;
 	//All indirect invocations of methods of the same class
-	private Set<String> methodInvocationsIndirectLocal;
+	private Map<String, Set<String>> methodInvocationsIndirectLocal;
 
 	public CKMethodResult(String methodName, String qualifiedMethodName, boolean isConstructor, int modifiers) {
 		this.methodName = methodName;
 		this.qualifiedMethodName = qualifiedMethodName;
 		this.isConstructor = isConstructor;
 		this.modifiers = modifiers;
+		this.isVisible = Modifier.isPublic(modifiers) || Modifier.isProtected(modifiers);
```

Comment:
shouldn't you count methods with the _default_ modifier too? Or are they not considered _visible_?

Reply:
To me, it is not clear from the definition, if the default modifier is considered visible or not. But, I think you are right and we should include it, as it is visible at least within the same package.